lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1276,6 +1283,7 @@ def _host_config(self):
             device_write_bps='device_write_bps',
             device_read_iops='device_read_iops',
             device_write_iops='device_write_iops',
+            pids_limit='pids_limit'
```

Comment:
Just a nit: can you add a trailing comma, so that the next time someone adds something, there's no need to modify this line?

Reply:
Done. Thank for pointing.